README.md

```diff
@@ -24,16 +24,17 @@ terraform apply
 ```
 
 Once the user is created you will need to update the
-[repository's secrets](https://github.com/cisagov/skeleton-packer/settings/secrets)
-with the new encrypted environment variables.
-
-```console
-terraform state show module.iam_user.aws_iam_access_key.key
-```
-
-Take the `id` and `secret` fields from the above command's output and create the
-`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables in the
-[repository's secrets](https://github.com/cisagov/skeleton-packer/settings/secrets).
+[repository's secrets](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)
+with the new encrypted environment variables. This should be done using the
+`terraform-to-secrets` tool available in the
+[development guide](https://github.com/cisagov/development-guide). Instructions
+for how to use this tool can be found in the
+[Project Setup README](https://github.com/cisagov/development-guide/tree/develop/project_setup).
+
+If you have appropriate permissions for the repository you view existing
```

Comment:
"...you view..." --> "...you can view..."?

Reply:
woops should be fixed in 599f2fa